code/modules/projectiles/guns/energy.dm

```diff
@@ -65,9 +65,13 @@
 	readout += "Standard models of this projectile weapon have <span class='warning'>[ammo_type.len]</span> mode\s"
 	readout += "Our heroic interns have shown that one can theoretically stay standing after..."
 	for(var/obj/item/ammo_casing/energy/for_ammo in ammo_type)
-		exam_proj = for_ammo.loaded_projectile
+		if(!for_ammo.projectile_type)
+			continue
+		exam_proj = new for_ammo.projectile_type() // Initial loaded projectile cannot be used because its initial value is null
```

Comment:
I'm wary of this, but I'll let it go. Can you at least qdel it once you're doing using it?

Reply:
Added. I'm wary of it as well, I just wasn't prepared for the strange way energy weapons handle whether or not a projectile is 'loaded' in its lens. I originally tried to fix this by just making sure there's always a projectile loaded into a KA, but after more loaded_projectile bugs came to light I figured this was the best way to do it